src/NuGet.Server/Web.config

```diff
@@ -43,7 +43,13 @@
     <!--
     Set enableFrameworkFiltering to true to enable filtering packages by their supported frameworks during search.
     -->
-    <add key="enableFrameworkFiltering" value="false"/>
+    <add key="enableFrameworkFiltering" value="false" />
+    
+	<!--
+	If value is setted to 0, there are no retention otherwise the retention will be setted to the defined value
+	-->
+	<add key="retentionRule" value="0" />
```

Comment:
>retentionRule [](start = 11, length = 13)

The naming is not super clear. How about: AllowedPackageVersionsCount 
And a comment: Set AllowedPackageVersionsCount to the number of versions of a package the server maintains. When the number of versions of a package exceeds this value, the oldest versions will be removed. Set AllowedPackageVersionsCount to 0 to disable retention policy.